tests/modules/programs/yazi/keymap-expected.toml

```diff
@@ -1,27 +1,27 @@
-[[input.keymap]]
-exec = "close"
+[[input.keymap_prepend]]
+run = "close"
```

Comment:
`run` and `on` need to be alphabetically ordered, else the `nix build -L .#test-yazi-settings` test fails.

This happens because Nix orders the attributes in alphabetical order, so the generated toml file is ordered as well, while the expected one is not, thus causing a conflict.

Reply:
done.